docs/query-classification-and-display.md

```diff
@@ -74,28 +74,3 @@ results it is supposed to find, i.e., how well it implements its
 on what we consider that rule to be. We generally try to sharpen our
 rules to focus on results that a developer might actually be interested
 in.
-
-## Which queries to run and display on LGTM
-
-The following queries are run:
-
-Precision:     | very-high | high    | medium  | low
----------------|-----------|---------|---------|----
-Error          | **Yes**   | **Yes** | **Yes** | No
-Warning        | **Yes**   | **Yes** | **Yes** | No
```

Comment:
Shouldn't we simply delete this entire file? The `Attributable Queries` is obsolete too, and the `and display` bit in  the title is only relevant to LGTM. Alternatively, you could of course rewrite this entire file to make it relevant to the way the default code-scanning suites work.